includes/fbs.debt.inc

```diff
@@ -5,31 +5,6 @@
  * Handles user debts.
  */
 
-/**
- * Fetches user debts.
- *
- * @param $account
- *   User object whose debts are fetched.
- *
- * @return array
- *   Set of user debts.
- */
-function fbs_debt_raw_list(stdClass $account) {
```

Comment:
This was specially added to be able to get debts list without any processing. Since invoking `fbs_debts_list()` also preloads entities which in case of use pages dramatically increases page load.